source/simulator/parameters.cc

```diff
@@ -657,6 +658,40 @@ namespace aspect
                            "the bilinear form is coercive, but not so large as to penalize "
                            "discontinuity at all costs.");
+        prm.declare_entry ("Use limiter for discontinuous temperature solution", "false",
+                           Patterns::Bool (),
+                           "Whether to apply the bound preserving limiter as a correction after having"
+                           "the discontinous temperature solution. Currently we apply this only to the"
+                           "temperature solution if the global max/min are already determined and defined"
+                           "This limiter keeps the discontinuous solution in the range given by"
+                           "Global temperature max or min.");
+        prm.declare_entry ("Use limiter for discontinuous composition solution", "false",
+                           Patterns::Bool (),
+                           "Whether to apply the bound preserving limiter as a correction after having"
+                           "the discontinous composition solution. Currently we apply this only to the"
+                           "compositional solution if the global max/min are already determined and defined"
+                           "This limiter keeps the discontinuous solution in the range given by"
+                           "Global composition max or min.");
+        prm.declare_entry ("Global temperature max", boost::lexical_cast<std::string>(std::numeric_limits<double>::max()),
+                           Patterns::Double (),
+                           "The maximum global temperature value that will be used in the bound preseving"
+                           "limiter for the discontinuous solutions from temperature advection fields."
+                           "Usually it's predetermined by the initial temperature field");
+        prm.declare_entry ("Global temperature min", boost::lexical_cast<std::string>(-std::numeric_limits<double>::max()),
+                           Patterns::Double (),
+                           "The minimum global temperature value that will be used in the bound preseving"
+                           "limiter for the discontinuous solutions from temperature advection fields."
+                           "Usually it's predetermined by the initial temperature field");
+        prm.declare_entry ("Global composition max", boost::lexical_cast<std::string>(std::numeric_limits<double>::max()),
+                           Patterns::List(Patterns::Double ()),
+                           "The maximum global composition values that will be used in the bound preseving"
+                           "limiter for the discontinuous solutions from composition advection fields."
+                           "Usually it's predetermined by the initial compositional field");
+        prm.declare_entry ("Global composition min", boost::lexical_cast<std::string>(-std::numeric_limits<double>::max()),
```

Comment:
Can you look through the other parameters to see whether we refer to something as `max` or `maximum`? I think we typically spell everything out.